tutorials/01_install.md

```diff
@@ -98,14 +98,14 @@ Build and install as follows:
 
 1. Clone the repository
   ```
-  git clone https://github.com/gazebosim/gz-gui -b ign-gui<#>
+  git clone https://github.com/gazebosim/gz-gui -b gz-gui<#>
```

Comment:
Can you add a note that if the version is > `gui7` then to use `ign-`?  It also might be good to add a link to this post: https://community.gazebosim.org/t/a-new-era-for-gazebo/1356

Reply:
Done. Could you take a look?

Reply:
Statement looks good, do you mind putting it closer to the beginning of this document, right above `Binary Install` heading here: https://github.com/gazebosim/gz-gui/blob/db15648c865a8ad5a7b39b5d83e0a6d5279e3bfd/tutorials/01_install.md?plain=1#L17-L18

I'd made sure to add a empty row before and afterwards so the `Note` is on it's own line, rather than part of the previous sentence.